### ISSUE TITLE
More rewards/medals

### ISSUE THREAD

**Issue by Grab11** [TheSil/base_enhanced#47]
Midair - airshot
Accuracy - finish a round with +50% accuracy
First Frag 
Headshot - headshot with sniper, pistol, blaster rifle, main repeater
Perfect - win round without taking damage
Force Master - capture flag with force boon
also printing rewards over players head.

**Comment by Grab11** [comment]
http://img.ctrlv.in/img/15/09/12/55f44b6f3d4be.png

**Comment by entdark** [comment]
:-1:

**Comment by NoahBennet** [comment]
:-1:  Very bad idea. If implemented there is a must to implement some kind of /awardtoggle command as well, so one can turn off all the awards Crab added, and have it the way it was about a half year ago.

**Comment by Grab11** [comment]
/cg_drawnewrewards - problem solved
Why you want to turn it off? Accuracy will be printed once, after match(hard to achieve +50% accuracy). Perfect will be achievable on duel only. First frag once a match, for one person only. Air saber almost never happens on CTF. Booner for one person only. It's hard to headshot in JKA. That will not be that spammy as you all think.
also spierdalaj sylor

**Comment by entdark** [comment]
To make it clear: I've downvoted for the assets, not the idea. Having it would be cool anyways.

**Comment by NoahBennet** [comment]
what does /cg_drawnewrewards do? Turns off all the shitty new awards you requested past 2 months? That would be cool.. If it turns off the rewards for defense, assist, cap too thats shit then.
Anyway i already explained you that if you want to have shitty quake live awards like this then try to play quake live instead of trying to convert jka to quake live.. Got it?

**Comment by Grab11** [comment]
1) u r shitty
2) defence and other rewards are from quake 3 but u don't have problem with them.
3) u r too bad for ql, I know.
@entdark i'm waiting for ur assets.

**Comment by deathsythe47** [comment]
Yeah, shame on Grab for wanting to create new content for a 15-year-old video game.

**Comment by NoahBennet** [comment]
Point out where i said it should NOT be implemented... He can do it, i am allright with that, but then make a toggle to it as well so ppl who wants to cherrypick which awards they want they can... Why is it a problem for you that i cherrypicked those 3 awards but dont want any else? I really dont need a award spam for every single sniper shot i make, thats pretty annoying, but i want the old ones i had for years and get to like them: eg. defense, assist, cap.
